compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.desktop.kt

```diff
@@ -155,6 +156,10 @@ internal class ComposeSceneMediator(
 
         override fun componentAdded(e: ContainerEvent) {
             val component = e.child
+            if (component !is SwingPanelContainer) {
```

Comment:
Could you describe the issue? I thought that this method is suppose to work for all components, not only `SwingPanelContainer`. Don't we need `addClipComponent` and `subscribeToMouseEvents` for them?

Reply:
It was fixed a while ago, so I decided to make PR to not forget about it. Converted to draft for now - I'll document/test-cover it once I have time